tests/test_filters.py

```diff
@@ -404,15 +440,22 @@ def _assert_column_definition(
         column_name,
         column_type,
         column_type_length,
-        column_attributes
+        column_attributes,
+        column_type_values=None
```

Comment:
`column_type_length` and `column_type_values` are both optional, `_assert_column_definition` can be called with both of them set to `None`. In such scenario it can be ambiguous to use either function (`_assert_column_definition_with_values` or `_assert_column_definition_with_length`)